src/address/mod.rs

```diff
@@ -115,8 +115,9 @@ impl<A: Actor> Addr<A> {
     #[inline]
     /// Send asynchronous message and wait for response.
     ///
-    /// Communication channel to the actor is bounded. if returned `Future`
-    /// object get dropped, message cancels.
+    /// Communication channel to the actor is bounded. If returned `Future`
```

Comment:
I'm not sure it is correct statement.
You can use this function correctly regardless of context.

The main point should be is that in `stopping` state your future would get dropped and it should be explicitly be told in Actor lifecycle https://actix.rs/actix/actix/trait.Actor.html#stopping